aviator.yml

```diff
@@ -12,6 +12,6 @@ spruce:
   to: aviator_pipeline.yml
 fly:
   name: orchestration-service
-  target: concourse
-  expose: false
+  target: aws-concourse
+  expose: true
```

Comment:
can you also add `check_creds: true`? I'll put a full explanation to the team

Reply:
Added